src/comparator.js

```diff
@@ -69,17 +71,86 @@ function normalise (entry: any) {
     'Customer Address Line 2': null
   }
 }
+
+let alternativeCountryNames = new Map([
+  ['GBR', 'UNITED KINGDOM'],
+  ['AU', 'AUSTRALIA'],
+  ['USA', 'UNITED STATES'],
+  ['CA', 'CANADA'],
+  ['THE NETHERLANDS', 'NETHERLANDS'],
+  ['HOLLAND', 'NETHERLANDS'],
+  ['TANZANIA', 'TANZANIA, UNITED REPUBLIC OF'],
+  ['RUSSIA', 'RUSSIAN FEDERATION'],
+  ['ANTIGUA', 'ANTIGUA AND BARBUDA'],
+  ['MACAU', 'MACAO'],
+  ['SOUTH KOREA', 'KOREA, REPUBLIC OF'],
+  ['IRAN', 'IRAN, ISLAMIC REPUBLIC OF'],
+  ['LAOS', 'LAOS, PEOPLE\'S DEMOCRATIC REPUBLIC OF'],
+  ['BRITISH VIRGIN ISLANDS', 'VIRGIN ISLANDS, BRITISH'],
+  ['KYRGYSTAN', 'KYRGYZSTAN'],
+  ['BOSNIA-HERZEGOVINA', 'BOSNIA AND HERZEGOVINA'],
+  ['CAPE VERDE ISLANDS', 'CAPE VERDE'],
+  ['FALKLAND ISLANDS', 'FALKLAND ISLANDS (MALVINAS)'],
+  ['VIETNAM', 'VIET NAM'],
+  ['TRINIDAD AND TOBAGO', 'TRINIDAD & TOBAGO']
+]);
```

Comment:
These are just the differences I saw in the comparator output.
To make this work on every possible case we should either see all the differences between [Zuora](https://knowledgecenter.zuora.com/BB_Introducing_Z_Business/D_Country%2C_State%2C_and_Province_Codes/B_State_Names_and_2-Digit_Codes) and [support-internationalisation](https://github.com/guardian/support-internationalisation/blob/master/src/main/scala/com/gu/i18n/CountryGroup.scala) country lists. Or just translate all the names in both lists into the 2 letter code.
In any case as long as the sf country selection is just a text field non standard names for the country are still possible